config/manager/kustomization.yaml

```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/infrawatch/smart-gateway-operator
-  newTag: 5.0.0
+  newTag: 5.0.0-dev5
```

Comment:
This keeps leaking in during development. Really will need to figure out the best way of dealing with this going forward.